lib/cib/cib_utils.c

```diff
@@ -338,37 +337,42 @@ cib_perform_op(const char *op, int call_options, cib_op_t * fn, gboolean is_quer
     );
     xml_accept_changes(scratch);
 
-    if (diff_cs == NULL) {
-        diff_cs = qb_log_callsite_get(__PRETTY_FUNCTION__, __FILE__, "diff-validation", LOG_DEBUG, __LINE__, crm_trace_nonlog);
```

Comment:
Note: debug was used here in the initialization step, but the later call to `crm_is_callsite_active()` always reset the priority to trace